src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,61 @@
+import { nanoid } from "nanoid";
+import PropTypes from "prop-types";
+import React, { ReactElement, ReactNode, useRef } from "react";
+
+import "./modal.scss";
+
+type Props = {
+  buttonRow?: ReactNode | ReactNode[] | null;
```

Comment:
If we're not limiting the button type then maybe this should just be `ReactNode | null;` and change the buttons in the example to use a fragment instead of passing an array (and having to worry about unique keys).

Reply:
good call